foursight_core/react/api/cognito.py

```diff
@@ -56,13 +70,14 @@ def _get_cognito_oauth_config_basic() -> dict:
     if not client_id:
         client_id = Gac.get_secret_value("COGNITO_CLIENT_ID")
     return {
-        "region": "us-east-1",
+        "region": AWS_REGION,
         "domain": domain,
-        "user_pool_id": user_pool_id,
-        "client_id": client_id,
+        "userpool": user_pool_id,
+        "client": client_id,
         "scope": ["openid", "email", "profile"],
         "connections": ["Google"],
-        "config": f"{COGNITO_BASE_URL}/{user_pool_id}/.well-known/openid-configuration"
+        "config": f"{AWS_COGNITO_SERVICE_BASE_URL}/{user_pool_id}/.well-known/openid-configuration",
```

Comment:
this could again be `self.BASE_URL`.

Reply:
Not sure I get your meaning; these are just functions, no class, no self.